appveyor/install-miniconda.ps1

```diff
@@ -76,8 +76,12 @@ $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
 
 # Conda config
 conda config --set always_yes true
-conda config --add channels astropy-ci-extras
 
+if (! $env:CONDA_CHANNELS) {
+   conda config --add channels astropy astropy-ci-extras

```

Comment:
@Cadair - But that logic wont work for the CONDA_CHANNELS variable, as we don't know how many elements are in it.